src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/DpsValidationserviceConstants.java

```diff
@@ -0,0 +1,18 @@
+package ca.bc.gov.open.pssg.rsbc.dps.dpsvalidationservice;
+
+/**
+ * @author nancymz
+ * Constants used in DPS Validation Service
+ */
+public class DpsValidationserviceConstants {
+    public static String VALIDOPEN_DFCMCASE_RESPONSE = "<GetValidOpenDFCMCase>" +
+            "<int>%s</int>" +
+            "<caseDesc>%s</caseDesc>" +
+            "</GetValidOpenDFCMCase>";
+
+    public static String VALIDOPEN_DFCMCASE_ERR_RESPONSE = "<GetValidOpenDFCMCase>" +
+            "<int>-2</int>" +
```

Comment:
I would substitute the -2 for %s and then use string.format to set the value. This way, we could use the err response string with any response code you wish for other situations.

Reply:
just a suggestion to follow Camel case convention for java class names.

Reply:
changed to a  camelCase name.  
String.format is used